sdk/Dockerfile

```
@@ -1,20 +1,57 @@
-FROM ghcr.io/cirruslabs/android-sdk:36
+FROM eclipse-temurin:21-jdk-noble
 
-LABEL org.opencontainers.image.source=https://github.com/cirruslabs/docker-images-flutter
 USER root
```

Hadolint (2.15.1)

[warning] Last USER should not be root

(DL3002)